packages/@glimmer/interfaces/lib/managers/modifier.d.ts

```diff
@@ -19,4 +19,6 @@ export interface ModifierManager<ModifierStateBucket> {
   installModifier(instance: ModifierStateBucket, element: Element, args: Arguments): void;
   updateModifier(instance: ModifierStateBucket, args: Arguments): void;
   destroyModifier(instance: ModifierStateBucket, args: Arguments): void;
+  getDebugName?(definition?: object): string;
```

Comment:
For this file, it's probably more appropriate to make this `factory: unknown` to match `createModifier`, since they are talking about the same thing. Not sure if it's missing a generic for the definition or if it's trying to be deliberately vague about it, but either way that problem predated your change so I'd just focus on matching those two here